Source/Fuse.Navigation/RoutePageProxy.uno

```diff
@@ -65,21 +65,23 @@ namespace Fuse.Navigation
 		}
 		List<Level> _levels = new List<Level>();
 		
+		/** `Init()` must be called to finish initialization. */
 		public RoutePageProxy( Visual source, ProgressUpdated progressUpdated )
 		{
 			_progressUpdated = progressUpdated;
-			Init(source);
+			_source = source;
 		}
 		
+		/** `Init()` must be called to finish initialization. */
 		public RoutePageProxy( Visual source, ActiveChanged activeChanged )
 		{
 			_activeChanged = activeChanged;
-			Init(source);
+			_source = source;
 		}
-		
-		void Init( Visual source )
+
+		/** Call after configuration is completed. This could start the actual registration and messaging */
+		public void Init()
```

Comment:
I'm a bit skeptical to this on a release-branch. It's technically an API-change, although not a very big one.

It looks to me like the setter for `RoutePageProxy.Path` could be used to skunk this in on the release-branch without touching the API, as all places we need this seems to call that setter right before where you add the Init()-call here...

That being said, the Init-approach seems much nicer. So that's probably what we want going forward.